docs/user/apirule-migration/01-82-migrate-allow-noop-no_auth-v1beta1-to-v2.md

```diff
@@ -22,6 +22,11 @@ Version `v1beta1` of APIRule is deprecated and scheduled for removal. Once the A
 
 This example demonstrates a migration from an APIRule `v1beta1` with **noop**, **allow**, and **no_auth** handlers to an APIRule `v2` with the **noAuth** handler.
 The example uses an HTTPBin service, exposing the `/anything`, `/headers`, and `/.*` endpoints. The HTTPBin service is deployed in its own namespace, with Istio enabled, ensuring the workload is part of the Istio service mesh.
+This example assumes that the targeted workload is only exposed by a single APIRule in version `v1beta1`.
+
+> [!WARNING]
+> If multiple APIRules target the same workload, you must perform an additional migration step to avoid traffic disruption. This step involves creating an additional, temporary AuthorizationPolicy before applying first migrated APIRule `v2` to allow traffic for workloads exposed by APIRules `v1beta1` not yet migrated to version `v2`. For detailed instructions, refer to the [Migrate multiple APIRules `v1beta1` targeting same workload](./01-90-migrate-multiple-apirules-targeting-same-workload.md) documentation before proceeding.
```

Comment:
```suggestion
> If multiple APIRules target the same workload, you must perform an additional migration step to avoid traffic disruption. This step involves creating an additional, temporary AuthorizationPolicy before applying the first migrated APIRule `v2`. For detailed instructions, refer to the [Migrate multiple APIRules `v1beta1` targeting same workload](./01-90-migrate-multiple-apirules-targeting-same-workload.md) documentation before proceeding.
```
I think it's enough to say "to avoid traffic disruption" in the warning, if someone needs more information, they can open the guide